source/blog/2016-06-14-rspec-3.5-has-been-released.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
In fact, I had to rename the file to get middleman to serve this blog post locally.  I guess it requires the dash form.

That suggests that you wrote this without seeing how it renders in middleman.  That's fine, but be sure to check how it renders in your browser.  I've seen plenty of cases where things did not render how you'd expect initially, and the manual check is important before we publish.